exe-unit/components/gsb-http-proxy/src/monitor.rs

```diff
@@ -0,0 +1,11 @@
+pub trait RequestsMonitor: Sync + Send + Clone {
```

Comment:
It will gone (or at least it will not be public), because Gsb Http Proxy service will hold handle to `MestricsService` (I will rename it to `Counters`), and update it by itself.